airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/components/SettingsView.test.tsx

```diff
@@ -17,9 +17,11 @@ jest.mock("hooks/services/useConnectionHook", () => ({
   useGetConnectionState: () => ({ state: null, globalState: null, streamState: null }),
 }));
 
-jest.mock("hooks/services/Analytics/useAnalyticsService", () => ({
-  useTrackPage: () => null,
-}));
+jest.mock("hooks/services/Analytics/useAnalyticsService", () => {
+  const analyticsService = jest.requireActual("hooks/services/Analytics/useAnalyticsService");
+  analyticsService.useTrackPage = () => null;
+  return analyticsService;
+});
```

Comment:
Had to make this change to prevent this test from failing since `AnalyticsProvider` was getting overridden.